components/default_products_list/index.tsx

```diff
@@ -0,0 +1,14 @@
+const DefaultProductsList = () => {
+  const numbers = [3, 1, 2, 5, 4];
+
+  return (
+    <>
+      <h3>I am Default list</h3>
+      {numbers.map((num) => (
+        <li>item-{num}</li>
+      ))}
+    </>
```

Comment:
Can you pick the exact product lists from the vercel example here
https://github.com/vercel/examples/blob/main/edge-functions/feature-flag-optimizely/pages/popular.tsx